src/data/Types.hpp

```diff
@@ -169,6 +169,11 @@ struct NFTsAndCursor {
     std::optional<ripple::uint256> cursor;
 };
 
+struct MPTHoldersAndCursor {
```

Comment:
Doxygen comments missing. 
It would also be great if you want to add one for the above NFT stuff that is missing them too but that's up to you 👍

Reply:
added